sql/core/src/test/resources/sql-tests/inputs/pgSQL/boolean.sql

```diff
@@ -23,7 +23,7 @@ SELECT false AS `false`;
 SELECT boolean('t') AS true;
 
 -- [SPARK-27931] Trim the string when cast string type to boolean type
-SELECT boolean('   f           ') AS `false`;
+SELECT boolean('   f           ') AS `true`;
```

Comment:
@wangyum I think it should be 'true'. Is it correct?

Reply:
It's `false`:
![image](https://user-images.githubusercontent.com/5399861/63177036-a8f95880-c079-11e9-9a8f-bcb9129a9942.png)

Reply:
Let me dig into this.